internal/testutils/helper.go

```diff
@@ -14,3 +34,85 @@ func PrintLogs(t *testing.T) {
 		t.Logf("Captured Logs: %s", string(logContent[:]))
 	}
 }
+
+// Utility method to initialize a PingOne SDK client for testing
+func GetPingOneSDKClientInfo(t *testing.T) *connector.SDKClientInfo {
+	t.Helper()
+
+	apiClientOnce.Do(func() {
+		// Grab environment vars for initializing the API client.
+		// These are set in GitHub Actions.
+		clientID := os.Getenv("PINGCTL_PINGONE_WORKER_CLIENT_ID")
+		clientSecret := os.Getenv("PINGCTL_PINGONE_WORKER_CLIENT_SECRET")
+		environmentId := GetEnvironmentID()
+		region := os.Getenv("PINGCTL_PINGONE_REGION")
+
+		if clientID == "" || clientSecret == "" || environmentId == "" || region == "" {
+			t.Fatalf("Unable to retrieve env var value for one or more of clientID, clientSecret, environmentID, region.")
+		}
+
+		apiConfig := &sdk.Config{
+			ClientID:      &clientID,
+			ClientSecret:  &clientSecret,
+			EnvironmentID: &environmentId,
+			Region:        region,
+		}
+
+		// Make empty context for testing
+		ctx := context.Background()
+
+		// Initialize the API client
+		client, err := apiConfig.APIClient(ctx)
+		if err != nil {
+			t.Fatalf(err.Error())
+		}
+
+		sdkClientInfo = &connector.SDKClientInfo{
+			Context:             ctx,
+			ApiClient:           client,
+			ExportEnvironmentID: environmentId,
+		}
+	})
+
+	return sdkClientInfo
+}
+
+func ValidateImportBlocks(t *testing.T, resource connector.ExportableResource, expectedImportBlocks *[]connector.ImportBlock) {
+	t.Helper()
+
+	importBlocks, err := resource.ExportAll()
+	if err != nil {
+		t.Fatalf("Failed to export %s: %s", resource.ResourceType(), err.Error())
+	}
+
+	// Check number of export blocks
+	expectedNumberOfBlocks := len(*expectedImportBlocks)
+	actualNumberOfBlocks := len(*importBlocks)
+	if actualNumberOfBlocks != expectedNumberOfBlocks {
+		t.Fatalf("Expected %d import blocks, got %d", expectedNumberOfBlocks, actualNumberOfBlocks)
+	}
+
+	// Make sure the importblocks match the expected import blocks
+	for index, importBlock := range *importBlocks {
+		expectedImportBlock := (*expectedImportBlocks)[index]
+
+		if !importBlock.Equals(expectedImportBlock) {
```

Comment:
Does the P1 API consistently return things in the same order?

Reply:
From my testing it has, but I think this may be worth fixing so that order doesn't matter for stability should that change.

Reply:
Updated this to expect a map for expectedImportBlocks